src/rpc/blockchain.h

```diff
@@ -10,8 +10,7 @@ class CBlockIndex;
 class UniValue;
 
 /**
- * Get the difficulty of the net wrt to the given block index, or the chain tip if
- * not provided.
+ * Get the difficulty of the net wrt to the given block index.
```

Comment:
From #13602: Can you drop the "to" here.

Reply:
Sure!